evo1/checkpoints.py

```diff
@@ -1,9 +1,28 @@
-from evo1.seq import MeadowFight, NoriaMines, OverworldToMeadow
+from engine.seq import SeqList
+from evo1.seq import (
+    CrystalCavern,
+    MeadowFight,
+    NoriaMines,
+    OverworldToCavern,
+    OverworldToMeadow,
+)
+
+
+class CavernCheckpoint(SeqList):
```

Comment:
Added a test segment here that starts just north of Papurika village and ends just after Kefka's Ghost.